### DISCUSSION TITLE
DSLite/ipv6 Support

### DISCUSSION THREAD

**Discussion by TheGor1lla** [onedr0p/cluster-template#604] [Q&A] [answered]
I'm behind a DSLite setup, but would like to access some apps from the internet.
Is it enough to manually change the A record to a AAAA record? Would it be useful to anyone integrating ipv6 into the setup portion?

**Comment by onedr0p** [comment]
I wish I was more knowledgeable with ipv6, you might find better support in our discord server. There's some people using ipv6, or trying to anyways. If you learn anything feel free to post back here with what you discovered. It could help other people out in the future!

**Comment by samip5** [comment, marked as answer]
IPv6 support was added to the template via https://github.com/onedr0p/flux-cluster-template/pull/855
